malta.ld

```diff
@@ -26,7 +26,7 @@ SECTIONS
     . = ALIGN(4096);
   } : text
 
-  .data ADDR(.text) + SIZEOF (.text):
+  .data :
```

Comment:
Looks good, I did not know about linker's behaviour w.r.t. orphaned sections. Please note that at some point we will have to investigate why FreeBSD's [ldscript.mips](http://bxr.su/FreeBSD/sys/conf/ldscript.mips) is so complex, i.e. if we're missing sth important.